CoffeeChat/Controller/LoginViewController.swift

```diff
@@ -98,7 +98,7 @@ extension LoginViewController: GIDSignInDelegate, MessageAlertViewDelegate {
             return
         }
 
-        if let email = user.profile.email, !(email.contains("@cornell.edu")) {
+        if let email = user.profile.email, !(email.contains("@cornell.edu")) && email != "cornellpearapp@gmail.com" {
```

Comment:
just curious why this email can't be cornellpearapp@gmail.com

Reply:
Here we are allowing only .cornell.edu emails or "cornellpearapp@gmail.com"